globus_action_provider_tools/testing/mocks.py

```diff
@@ -39,7 +39,7 @@ def mock_introspect_token(*args, **kwargs):
     patch(
         "globus_action_provider_tools.authentication.AuthState.identities",
         new_callable=PropertyMock,
-        return_value=frozenset("MOCK_USER"),
+        return_value=frozenset(["MOCK_USER"]),
```

Comment:
oops, that seems like a good catch. Did this end up as a set of `{'M', 'O', 'C', ...}` otherwise. Hate those strings are iterable errors.

Reply:
Exactly, I'm sad it masqueraded around as long as it did to begin with...